src/sidebar/util/disable-opener-for-external-links.js

```diff
@@ -17,8 +17,10 @@
  */
 export default function disableOpenerForExternalLinks(root) {
   root.addEventListener('click', event => {
-    if (event.target.tagName === 'A') {
-      const linkEl = event.target;
+    const target = /** @type {HTMLElement} */ (event.target);
+
+    if (target.tagName === 'A') {
+      const linkEl = /** @type {HTMLAnchorElement} */ (target);
```

Comment:
This is an example of using `@type` to help TypeScript figure out when the surrounding logic (the `target.tagName` check) implies something about the types of the values.